webhook/resources/volume/mutator.go

```diff
@@ -189,8 +191,13 @@ func (v *volumeMutator) Create(request *admission.Request, newObj runtime.Object
 			return nil, werror.NewInvalidError(fmt.Sprintf("get invalid size for volume %v: %v", backup.Status.VolumeSize, err), "")
 		}
 
-		moreLabels[types.LonghornLabelBackupVolume] = bvName
+		moreLabels[types.LonghornLabelBackupVolume] = remoteBVName
+	}
+	if backupTargetName == "" {
+		backupTargetName = types.DefaultBackupTargetName
 	}
```

Comment:
When is `backupTargetName` empty("")?

Reply:
When users create a volume without assigning a value to the `Spec.BackupTargetName`.